web/viewer.css

```diff
@@ -494,34 +468,29 @@ html[dir='ltr'] .doorHangerRight:before {
   background-color: rgb(255, 102, 102);
 }
 
-html[dir='ltr'] #toolbarViewerLeft {
-  margin-left: -1px;
```

Comment:
Also, removing this rule (and the corresponding RTL rule below) actually moved the left side of the toolbar *one* pixel.